src/CMakeLists.txt

```diff
@@ -104,7 +104,7 @@ set(static_depends)
 
 # Target link libraries
 if(SUPPORT_OMP)
-  if(WIN32)
+  if(MSVC)
```

Comment:
We often use `CXX=clang++` on Windows. I'm not sure, but I think `MSVC` would be false for that case, while `WIN32` would be true. Perhaps `WIN32 AND NOT MINGW` would be more appropriate?

Reply:
Good point. 👍
I'll wait for feedback on @jsandham's comment and either fix that or remove that condition entirely.

I seem to recall that there were issues in CMake when using OpenMP while targeting MSVC with `clang++`. A workaround at the time was to use `clang-cl` instead.
That issue should be fixed in newer versions of CMake.

Which version of CMake do you use?

Reply:
The condition should more specifically the situations in which this work-around is needed in the latest iteration of the PR.

Is that change acceptable?